CHANGELOG.md

```diff
@@ -2,31 +2,26 @@
 
 ### 1.5 (Dec 11, 2015)
 
--   Pipeline support
--   Requires 1.609 or higher of Jenkins.
+- Support for [**Pipeline**](https://github.com/jenkinsci/pipeline-plugin).
```

Comment:
I would not remove the information about the minimum core version requirement 1.609 for this version of the plugin.

Reply:
Thanks! It makes sense to let users known about it.